.eslintignore

```diff
@@ -1 +1,2 @@
-helix-importer-ui
\ No newline at end of file
+helix-importer-ui
+.eslintrc.js
```

Comment:
Do we need this? Is there a lint error because of this?

Reply:
Earlier, It was showing lint error for this fille so i added. But now, when i checked, it seems to be fine so removing it.